sandbox/public/index.html

```diff
@@ -71,7 +71,8 @@
           }
         },
         data: {
-          nonXdmKey: "nonXdmValue"
+          nonXdmKey: "nonXdmValue",
+          name: "home"
```

Comment:
Can you add a comment that this key/value is for Audience Manager?

Reply:
Done @jonsnyder